src/Spryker/Zed/EventBehavior/Business/EventBehaviorFacade.php

```diff
@@ -19,11 +20,11 @@ class EventBehaviorFacade extends AbstractFacade implements EventBehaviorFacadeI
      *
      * @api
      *
-     * @return void
+     * @return \Generated\Shared\Transfer\EventTriggerResponseTransfer
      */
-    public function triggerRuntimeEvents()
+    public function triggerRuntimeEvents(): EventTriggerResponseTransfer
     {
-        $this->getFactory()->createTriggerManager()->triggerRuntimeEvents();
+        return $this->getFactory()->createTriggerManager()->triggerRuntimeEvents();
```

Comment:
This would be a hard BC break if the typehint (void) would already have been in place.
With the current change, it seems to be a bit in the gray area :)
I guess it is fine as long as no one has implemented/customized this method on project level.